src/gui/properties/piecesbar.cpp

```diff
@@ -286,7 +286,10 @@ void PiecesBar::showToolTip(const QHelpEvent *e)
     }
     else {
         stream << simpleToolTipText();
-        stream << '\n' << tr("Hold Shift key for detailed information");
+        if (showDetailedInformation) // metadata are not available at this point
+            stream << '\n' << tr("Wait until metadata become available to see detailed information");
+        else
+            stream << '\n' << tr("Hold Shift key for detailed information");
```

Comment:
Because we said them in simple tooltip that they cab hold `Shift` to get detailed information. So we have to explain why this does not work.